ln-gateway/src/ln.rs

```diff
@@ -66,3 +70,66 @@ impl LnRpc for Mutex<cln_rpc::ClnRpc> {
         }
     }
 }
+
+/// Control the behavoir of the LnRpc by exposing the same interface but attach configurations and
+/// methods to it.
+pub struct LnRpcAdapter {
+    // add more configuration
```

Comment:
`TODO: `?

Reply:
no more of a "if you need to alter behavior do it here" but I'll remove that comment its a bit pointless